frontend/public/components/storage-class-form.tsx

```diff
@@ -55,7 +57,111 @@ export class StorageClassForm_ extends React.Component<
     this.previousName = '';
   }
 
-  storageTypes = Object.freeze({
```

Comment:
Leave the static list of storage types as a frozen object constant, then add the dynamic ones into state.

Reply:
Ack